.gitignore

```diff
@@ -30,10 +30,13 @@ pip-log.txt
 
 # Unit test / coverage reports
 .coverage
+.coverage.integration.*
+.coverage.unit
```

Comment:
Should be `.coverage.unit.*`. Other than that, LGTM.

Reply:
Here's what I'm seeing...  ```Untracked files:
  (use "git add <file>..." to include in what will be committed)

	.coverage.integration.st2actions
	.coverage.integration.st2api
	.coverage.integration.st2auth
	.coverage.integration.st2client
	.coverage.integration.st2common
	.coverage.integration.st2debug
	.coverage.integration.st2reactor
	.coverage.unit
	htmlcov/```